### ISSUE TITLE
Survival Cox Proportional Hazards & Competing Risks Adjustment

### ISSUE THREAD

**Issue by shaunporwal** [statsmodels/statsmodels#9421]
Hi!

I was hoping to implement a competing risks estimator in my python package dcurves. I was wondering if this package would build out its survival modules to incorporate cox proportional hazards/competing risks estimator functionalities? 

I know that there's another package called lifelines which handles survival statistics in greater depth, but I'd much rather just use the statsmodels package for my survival stats needs.

Thanks,
Shaun

**Comment by josef-pkt** [comment]
do you mean box-cox with more than one hazard?

I never looked at that. Do you have a main reference for it?

**Comment by shaunporwal** [comment]
Exactly, and that's the correct paper detailing the Fine & Gray method!
